sunpy/net/vso/exceptions.py

```diff
@@ -2,7 +2,6 @@ class NoData(Exception):
     """
     Risen for callbacks of VSOClient that are unable to supply information for the request.
     """
-    pass
```

Comment:
I don't think this should be removed...

Reply:
it isn't needed the docstring is the body of the class.